common/src/main/java/net/adoptopenjdk/icedteaweb/BasicFileUtils.java

```diff
@@ -0,0 +1,87 @@
+package net.adoptopenjdk.icedteaweb;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.io.Reader;
+import java.io.Writer;
+
+public class BasicFileUtils {
+    /**
+         * Method to save String as file in UTF-8 encoding.
+         *
+         * @param content which will be saved as it is saved in this String
+         * @param f file to be saved. No warnings provided
+         * @throws IOException if save fails
+         */
+        public static void saveFile(String content, File f) throws IOException {
+            saveFile(content, f, "utf-8");
+        }
+
+    /**
+         * Method to save String as file in specified encoding/.
+         *
+         * @param content which will be saved as it is saved in this String
+         * @param f file to be saved. No warnings provided
+         * @param encoding of output byte representation
+         * @throws IOException if save fails
+         */
+        public static void saveFile(String content, File f, String encoding) throws IOException {
```

Comment:
use `Charset` instead of String for encoding

Reply:
done